authors/apps/authentication/serializers.py

```diff
@@ -13,7 +21,19 @@ class RegistrationSerializer(serializers.ModelSerializer):
     password = serializers.CharField(
         max_length=128,
         min_length=8,
-        write_only=True
+        write_only=True, validators=[password_validator]
+    )
+    email = serializers.EmailField(
+                                   max_length=30,
+                                   validators=[UniqueValidator(
+                                    queryset=User.objects.all(),
+                                    message="Email already exists, please login instead")],
+                                   )
+    username = serializers.CharField(
+        max_length=30,
+        validators=[UniqueValidator(
+            queryset=User.objects.all(),
+            message="Username already exists, please enter another one")],
```

Comment:
I think that having the message just as Username already exists is enough. The part that says please enter another one is self-explanatory.

Reply:
Hey, @lennykioko The task required descriptive messages that give users an action to take. People might not understand it the way you do as a dev.